packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -248,12 +248,15 @@ const main = async () => {
                 process.exit(1)
             }
 
-            execSync(`${babelNode} ${inspect ? '--inspect' : ''} ${entrypoint}`, {
-                env: {
-                    ...process.env,
-                    ...(noHMR ? {HMR: 'false'} : {})
+            execSync(
+                `${babelNode} ${inspect ? '--inspect ' : '--extensions ".ts"'} ${entrypoint}`,
```

Comment:
why don't we need --extension when inspect is off? would it work if I run `npm run start:inspect`?

Reply:
Also, does `--extensions` replace or extend the default values? Do we need to do `--extendsions ".js,.ts,.jsx,.tsx"`?

Bonus points if this PR includes a change let `pwa-kit-dev start` pass through arbitrary additional arguments to `babel-node`.

Reply:
@wjhsf suggestion sounds good 👍